_data/projects/AV_WebsiteOne.yml

```diff
@@ -0,0 +1,14 @@
+name: AgileVentures WebsiteOne
+desc: A website for AgileVentures.org - a platform for online collaboration and crowdsourced project development.
+site: https://github.com/AgileVentures/WebsiteOne
+tags:
+- rails
+- ruby
+- tdd
+- bdd
+- HTML
+- CSS
+- javascript
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/AgileVentures/WebsiteOne/labels/up-for-grabs
```

Comment:
it'd be great to have some issues tagged with `up-for-grabs` before I merge this in...